usmqe_tests/api/gluster/conftest.py

```diff
@@ -100,18 +125,31 @@ def invalid_volume_configuration(request):
     return request.param
 
 
-@pytest.fixture
-def valid_volume_name():
-    """
-    Generate valid volume name defined in usm.ini.
-    """
-    return pytest.config.getini("usm_volume_name")
-
-
 # TODO(fbalak) as `./,!@##$%^&*()__{}|:';/<*+>)(` as parameter
 @pytest.fixture(params=[None])
 def invalid_volume_name(request):
     """
     Generate invalid volume name.
     """
     return request.param
```

Comment:
no part of this PR, but there should not be `return` in fixtures, `yield` is used instead

Reply:
It is possible to use ```return``` in case of tear-down absence, check https://docs.pytest.org/en/latest/fixture.html